kedro/contrib/idempotent/idempotent_state_storage.py

```diff
@@ -1,52 +1,71 @@
-import os
-from datetime import datetime
+from uuid import uuid4
 from typing import List
 
 
 class IdempotentStateStorage:
+    def __init__(self, run_id_state=None, input_state=None, nodes_have_been_run=None):
+        if run_id_state is None:
+            run_id_state = {
+                # "ds1": "qwer-qwer-wqer",
+                # "ds2": "adsf-asdf-adsf"
+            }
+        if input_state is None:
+            input_state = {
+                # "node1": {},
+                # "node2": {
+                #     "ds1": "qwer-qwer-qwer"
+            }
+        if nodes_have_been_run is None:
```

Comment:
Remove this state, and use `run_id_state` to determine if it has already been run, instead.